src/inc-lib-init.php

```diff
@@ -1744,12 +1750,13 @@ function lovd_getVariantInfo ($sVariant, $sTranscriptID = '', $bCheckHGVS = fals
                                         'insertions or deletion-insertions' :
                                         'variants which took place within a range') . '.';
 
-
     } elseif ($aVariant['suffix']) {
-        if (!in_array($aVariant['type'], array('ins', 'delins')) &&
-            (!isset($aResponse['messages']['IPOSITIONRANGE'])) || $aResponse['type'] == 'repeat') {
-            // If the variants are of a type that is not ins or delins, they should not have a suffix,
-            //  except if they were given a position range. Repeats should never be given a suffix.
+        if ($aResponse['type'] == 'repeat'
+            || (!in_array($aVariant['type'], array('ins', 'delins'))
+                && !isset($aResponse['messages']['IPOSITIONRANGE']))) {
```

Comment:
@loeswerkman, could you please check if I did this right? The `if()` condition was mixing `&&` and `||` without parentheses, and I think this is what you meant. I updated the order as well to aid in readability.